sdk/confidentialledger/Azure.Security.ConfidentialLedger/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0 (2023-08-03)
+
+### Bugs Fixed
+
+- Create a custom ResponseClassifier to allow `HttpStatusCode.NotFound` to be retriable. These errors may occur in scenarios where there is an unexpected loss of session stickiness when the connected node changes and transactions have not been fully replicated.
```

Comment:
I would reword this to a customer focused scenario. For example:

"Service calls that result in a `HttpStatusCode.NotFound` status will now be retried by default. This is to handle scenarios where ...."

Reply:
Updated the wording.